pkgs/by-name/ab/aba/package.nix

```diff
@@ -16,29 +16,27 @@ rustPlatform.buildRustPackage {
     owner = "~onemoresuza";
     repo = "aba";
     rev = version;
-    hash = "sha256-Sz9I1Dw7wmoUPpTBNfbYbehfNO8FK6r/ubofx+FGb04=";
-    domain = "sr.ht";
+    hash = "sha256-2zVQNchL4DFh2v2/kwupJTBSmXiKqlxzUMrP9TbfCMs=";
   };
 
-  cargoSha256 = "sha256-Ihoh+yp12qN74JHvJbEDoYz+eoMwPOQar+yBEy+bqb0=";
+  cargoSha256 = "sha256-YhSzbfcEIJjKWlyYq1lK70qt4f/Z71n7hgaaZ/D/U80=";
 
   nativeBuildInputs = [
     just
     scdoc
   ];
 
   postPatch = ''
-    # Suppress messages of command not found. jq is not needed for the build, but just calls it anyway.
-    sed -i '/[[:space:]]*|[[:space:]]*jq -r/s/jq -r .*/: \\/' ./justfile
-    # Let only nix strip the binary by disabling cargo's `strip = true`, like buildRustPackage does.
+    # Let only nix strip the binary by disabling cargo's `strip = true`, like
+    # buildRustPackage does when not using just's setup hooks.
     sed -i '/strip[[:space:]]*=[[:space:]]*true/s/true/false/' ./Cargo.toml
   '';
 
   preBuild = ''
     justFlagsArray+=(
       PREFIX=${builtins.placeholder "out"}
       MANIFEST_OPTS="--frozen --locked --profile=release"
-      INSTALL_OPTS=--no-track
+      INSTALL_OPTS="--no-track"
     )
   '';
```

Comment:
Any reason to this to be set in `preBuild` attribute instead of `justFlags`attribute like: https://github.com/nixos/nixpkgs/blob/master/pkgs/applications/window-managers/cosmic/settings/default.nix#L59-L66

Reply:
None. Gonna make a force-push with `justFlags` instead.

Reply:
This is surprising, spaces should have worked in `justFlags` I imagine (unless the hook is doing something horrible wrong or someething).

Anyway, I don't have a strong preference of having all in `justFlagsArray` or a mix of `justFlags` and `justFlagsArray`, but I definitely think we need a comment here at least explaining why we are not using `justFlags` here.

Reply:
I've added a comment explaining why `justFlags` is used instead of `justFlagsArray`.